doc/getting_started/build_sw.md

```diff
@@ -121,9 +121,9 @@ You may find it useful to use wildcards to build/test all targets in the OpenTit
 If a a target (a test or build artifact) relies on optional parts of the "Getting Started" guide they should be tagged so they can be filtered out and users can `bazelisk.sh test //...` once they filter out the appropriate tags.
 We maintain or use the following tags to support this:
 * `broken` is used to tag tests that are committed but should not be expected by CI or others to pass.
-* `cw310_test_rom`, `cw310_rom`, and `cw310_other` are used to tag tests that depend on a correctly setup cw310 "Bergen Board" to emulate OpenTitan.
-  The `cw310` prefix may be used in `--test_tag_filters` to enable concise filtering.
-  Loading the bitstream is the slowest part of the test, so these tags can group tests with common bitstreams to accelerate the tests.
+* `cw310`, `cw310_test_rom`, and `cw310_rom` are used to tag tests that depend on a correctly setup cw310 "Bergen Board" to emulate OpenTitan.
```

Comment:
I think we could make this text more prescriptive. I'll write the decision procedure in pseudocode — is this right?

```
if target needs a cw310 board:
  target.add_tag('cw310')
  if target depends on //hw/bitstream:rom:
    target.add_tag('cw310_rom')
  elif target depends on //hw/bitstream:test_rom:
    target.add_tag('cw310_test_rom')
```

Reply:
There are differences of opinion regarding cw310_rom, so this is what I can write without resolving disagreements. We'll try to make progress at the SW WG next Tuesday.